Assets/Engine/Inventory/Extensions/Hands.cs

```diff
@@ -98,10 +98,13 @@ private void Awake()
             // Because we just make calls to GetSlot, which is set pre-Awake, this is safe.
             handSlots = new int[2] { -1, -1 };
             for (int i = 0; i < handContainer.Length(); ++i) {
-                if (handContainer.GetSlot(i) == Container.SlotType.LeftHand)
+                if (handContainer.GetFilter(i).Hash == Animator.StringToHash("LeftHand".ToUpper()))
                     handSlots[0] = i;
-                else if (handContainer.GetSlot(i) == Container.SlotType.RightHand)
+                else if (handContainer.GetFilter(i).Hash == Animator.StringToHash("RightHand".ToUpper()))
+                {
                     handSlots[1] = i;
+                    Debug.Log(handContainer.GetFilter(i).name + "/" + handContainer.GetFilter(i).Hash);
```

Comment:
Seems like this Log is left over from testing.

Reply:
most of this problems are cause 2 filers were not merged :v but i'll fix the rest as well, and the filter should never be null